lib/nexpose/site.rb

```diff
@@ -241,16 +216,19 @@ def remove_asset(asset)
       begin
         # If the asset registers as a valid IP, remove as IP.
         IPAddr.new(asset)
-        remove_ip(asset)
+        @assets = assets.reject { |ip| ip == IPRange.new(asset) }
```

Comment:
> Use the dangerous version of [`Array#reject!`](http://www.ruby-doc.org/core-2.2.0/Array.html#method-i-reject-21) since we're just assigning the result to the result of `Arrary#reject`'s to the same variable.